copulas/univariate/gaussian.py

```diff
@@ -73,6 +78,9 @@ def cumulative_distribution(self, X):
             np.ndarray: Cumulative density for X.
         """
         self.check_fit()
+        if self.constant_value:
```

Comment:
Instead of checking for `self.constant_value` every time, I would just replace the instance method inside the `fit` method.

```
def fit():
    self.constant_value = self._get_constant_value(X)
    if self.constant_value is not None:
        self.cumulative_distribution = self._constant_cumulative_distribution
        # etc.
```